instrumentation/pekko/pekko-http-1.0/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/pekkohttp/v1_0/server/route/TapirPekkoHttpServerRouteInstrumentationModule.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.pekkohttp.v1_0.server.route;
+
+import static java.util.Arrays.asList;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.javaagent.extension.instrumentation.InstrumentationModule;
+import io.opentelemetry.javaagent.extension.instrumentation.TypeInstrumentation;
+import io.opentelemetry.javaagent.extension.instrumentation.internal.ExperimentalInstrumentationModule;
+import java.util.List;
+
+@AutoService(InstrumentationModule.class)
+public class TapirPekkoHttpServerRouteInstrumentationModule extends InstrumentationModule
```

Comment:
Would it make sense to override `classLoadMatcher` here to prevent _both_ this and `PekkoHttpServerRouteIntrumentation` from "firing" in the situation of tapir/pekko?

Reply:
@laurit I did some exploring here and there doesn't seem to be "double-instrumentation"--I inspected the general bytebode in the case where Tapir is on the classpath and it looks correct.

Reply:
as far as I can tell `RouteConcatenationInstrumentation` is applied twice, removed it from the tapir module

Reply:
Apologies for that. I'm curious: how did you determine that?